package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aerospike",
-  "version": "4.0.1",
+  "version": "5.0.0",
```

Comment:
4.1.0 or 5.0.0? Are there any backwards incompatible changes that warrant a major version jump? I thought this PR only adds new features, so a minor version jump should be sufficient.

Reply:
major version needs to be bumped up because of feature drop - "PredicateExpression filter Replaced by new Expressions Filter".